web/layout_attributes/using_layout_attributes/my_horizontal_element.dart

```diff
@@ -0,0 +1,11 @@
+@HtmlImport('my_horizontal_element.html')
+library my_horizontal_element;
+
+import 'package:web_components/web_components.dart' show HtmlImport;
+import 'package:polymer/polymer.dart';
+import 'package:polymer_elements/iron_flex_layout.dart';
```

Comment:
should be 
`import 'package:polymer_elements/iron_flex_layout/classes/iron_flex_layout.dart';`, same for `my_vertical_element.dart`